og.module

```diff
@@ -150,9 +150,8 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
     }
 
     $handler = Og::getSelectionHandler($field_definition);
-
-    if ($handler->getReferenceableEntities()) {
-      return AccessResult::neutral();
+    if ($handler->countReferenceableEntities()) {
+      return AccessResult::allowed();
```

Comment:
Thought about it more. (1) sounds like going back to D6, but in a bad way. (2) sounds ok, but we won't gain much.

In short, I think we should keep those features in.